A6-client-retries.md

```diff
@@ -148,6 +156,13 @@ If server pushback that specifies not to retry is received in response to a hedg
 
 Hedged requests should be sent to distinct backends, if possible. To facilitate this, the gRPC client will maintain a list of previously used backend addresses for each hedged RPC. This list will be passed to the gRPC client's local load-balancing policy. The load balancing policy may use this information to send the hedged request to an address that was not previously used. If all available backend addresses have already been used, the load-balancing policy's response is implementation-dependent.
 
+##### Validation of hedgingPolicy
+
+If `hedgingPolicy` is specified in a service config choice, the following validation rules apply:
+1. `maxAttempts` should be specified and should be a JSON integer value greater than 1. Values greater than 5 are treated as 5 without being considered a validation error.
+2. `hedgingDelay` is an optional field but if specified should follow the JSON representation of proto3 Duration type.
+3. `nonFatalStatusCodes` is a required field and follows the same validation rules as `retryableStatusCodes` in `retryPolicy`.
```

Comment:
Please copy the text instead of referring back to it.

Reply:
done